en/sim_gazebo_gz/vehicles.md

```diff
@@ -52,6 +52,16 @@ The camera cannot yet be used to stream video or for image capture in QGroundCon
 [PX4-Autopilot#22563](https://github.com/PX4/PX4-Autopilot/issues/22563) can be used to track the additional work needed to fully enable these use cases.
 :::
 
+### X500 Quadrotor with 2D LIDAR
+
+This model have a 2D LIDAR attached, modelled on the Hokuyo TM-30LX.
```

Comment:
This all looks excellent. A few follow ons:

1. did you mean [UTM-30LX](https://hokuyo-usa.com/products/lidar-obstacle-detection/utm-30lx) - google didn't find anything at all on TM-30LX.
2. Is this a lidar that we support in PX4? I can't find it here: https://docs.px4.io/main/en/sensor/rangefinders.html#distance-sensors-rangefinders
3. Also in that doc we have Gazebo Classic simulation section - https://docs.px4.io/main/en/sensor/rangefinders.html#gazebo-classic-simulation - Would it be possible for you to add a few words about simulation with Gazebo in its own section?

Reply:
PS You'll note there are more images in this PR than you had. I shrank all images linked in this folder (losslessly) so they just happen to appear as part of this.

Reply:
1. yea, youre right. thats a copypasta error. 
2. not that i can see. the only similar one we support is the [Lightware SF45/B](https://lightwarelidar.com/shop/sf45-b-50-m/) then it might make sense to remove the specific model. not sure of the best practices here.
3. currently the rotating Lightware lidar is also in this section, but we could add a subsection...
4. i agree that it would make sense to add a gazebo section to rangefinders, but i'm not sure if i'm the right person, as i have limited expierience creating my own models.
5. see (2.)

Reply:
Thanks @chfriedrich98 

> 2. not that i can see. the only similar one we support is the [Lightware SF45/B](https://lightwarelidar.com/shop/sf45-b-50-m/) then it might make sense to remove the specific model. not sure of the best practices here.
> 3. currently the rotating Lightware lidar is also in this section, but we could add a subsection...

For now we leave both of these as is - can't do any harm and I'm not sure we'd be improving things.

I've added comments for 4 and 5 in other places - mostly for you to help me understand "what can we do now we have this model".